packages/svelte/src/routes/+page.svelte

```diff
@@ -0,0 +1,3 @@
+<h1>Welcome to your library project</h1>
```

Comment:
Is this file required?

Reply:
Honestly not sure, I guess not. I'll look into it